inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -881,10 +889,9 @@ mortality_rate(deaths, population, multiplier = 10000, mergeCI = TRUE) %>%
   kable(digits = 2)
 ```
 
-To produce a crude mortality rate per 10,000 people per day, use the folowing 
+To produce a crude mortality rate attributable to measles per 10,000 people per day, use the folowing 
 code chunk. 
-This assumes that you are capturing every death in your population and that 
-your population remains stable over the time period of interest. 
+This assumes that you are capturing every measles death in your population and that your population remains stable over the time period of interest. 
```

Comment:
These linebreaks were actually in there on purpose  (makes your rmd easier to read and dont show up as line breaks in your word doc). 
To show a line indicating 80 characters - go to globaloptions > code > displace and click show margin